docs/rst/tutorials/core/qos/mutable/mutable.rst

```diff
@@ -0,0 +1,617 @@
+.. _tutorials_qos_mutable_mutable:
+
+Change mutable QoS through get native API
+=========================================
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+Background
+----------
+
+Fast DDS over Vulcanexus offers the possibility of not only configuring the QoS policy when creating ROS nodes, but it is also possible to modify them (some of them) after having created the ROS node.
+The QoS that allow their modification after that, are called mutable QoS.
+
+`Here <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/core/policy/policy.html>`_ you can find reference of all supported QoS Policy, and you can check wether any QoS is mutable in these links:
+
+    * `DataReader QoS <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/subscriber/dataReader/dataReader.html?highlight=mutable#datareaderqos>`_
+    * `DataWriter QoS <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/publisher/dataWriter/dataWriter.html?highlight=mutable#datawriterqos>`_
+    * `Topic QoS <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/topic/topic/topic.html?highlight=mutable#topicqos>`_
+    * `DomainParticipant QoS <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/domain/domainParticipant/domainParticipant.html?highlight=mutable#domainparticipantqos>`_
+    * `Publisher QoS <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/publisher/publisher/publisher.html?highlight=mutable#publisherqos>`_
+    * `Subscriber QoS <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/subscriber/subscriber/subscriber.html?highlight=mutable#subscriberqos>`_
+
+Another :ref:`tutorial <tutorials_qos_ownership_ownership>` explained the use of the Ownership and Ownership Strength QoS (see `Ownership QoS Policy <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/core/policy/standardQosPolicies.html#ownershipqospolicy>`_) and how to configure it within the ROS 2 talker/listener demo.
+This tutorial will show how to change Ownership Strength QoS in runtime, after all nodes have been already deployed.
+We will see how three nodes (one subscriber and two publishers) interact: After creation the subscriber will only be receiving data from the publisher with bigger ownership strength (corresponding to exclusive ownership QoS), and after that we will change the ownership strength of the other one to become bigger, thus the subscriber will start to show the data of the latter.
```

Comment:
There are some `we` and `you` here and there that I'd try to avoid

Reply:
Done